### ISSUE TITLE
Message:Sequence contains no matching element

### ISSUE THREAD

**Issue by Tychodewaard** [2sic/2sxc#1210]
I'm submitting a bug report.

I have a DNN instance where 2SXC (8.2) was already on the 804 install.
Adding the module to a page, resolved in a application error. 
Removing the module from the page, made the page function properly again.

I upgraded 2SXC to 9.2 and the installer won't go through.
Even after 12 times, complete upgrade, the upgrade is not completed.

The event viewer says:
Message:Sequence contains no matching element

```
at System.Linq.Enumerable.Single[TSource](IEnumerable`1 source, Func`2 predicate)
   at ToSic.Eav.Repository.Efc.Parts.DbImport.ImportAttributeSet(ImpAttrSet impAttrSet) in C:\Projects\eav-server\ToSic.Eav.Repository.Efc\Parts\DbImport.cs:line 155
   at ToSic.Eav.Repository.Efc.Parts.DbImport.ImportSomeAttributeSets(IEnumerable`1 newAttributeSets) in C:\Projects\eav-server\ToSic.Eav.Repository.Efc\Parts\DbImport.cs:line 118
   at ToSic.Eav.Repository.Efc.Parts.DbImport.<>c__DisplayClass11_0.b__0() in C:\Projects\eav-server\ToSic.Eav.Repository.Efc\Parts\DbImport.cs:line 81
   at ToSic.Eav.Repository.Efc.Parts.DbVersioning.QueueDuringAction(Action action) in C:\Projects\eav-server\ToSic.Eav.Repository.Efc\Parts\DbVersioning.cs:line 52
   at ToSic.Eav.Repository.Efc.Parts.DbImport.ImportIntoDb(IEnumerable`1 newAttributeSets, IEnumerable`1 newEntities) in C:\Projects\eav-server\ToSic.Eav.Repository.Efc\Parts\DbImport.cs:line 112
   at ToSic.Eav.Apps.ImportExport.XmlImportWithFiles.ImportXml(Int32 zoneId, Int32 appId, XDocument doc, Boolean leaveExistingValuesUntouched)
   at ToSic.SexyContent.Installer.V7.Version070303() in C:\Projects\2SexyContent\Web\DesktopModules\ToSIC_SexyContent\SexyContent\Installer\V7.cs:line 346
   at ToSic.SexyContent.Installer.InstallationController.UpgradeModule(String version) in C:\Projects\2SexyContent\Web\DesktopModules\ToSIC_SexyContent\SexyContent\Installer\InstallationController.cs:line 197
   at ToSic.SexyContent.Installer.InstallationController.FinishAbortedUpgrade() in C:\Projects\2SexyContent\Web\DesktopModules\ToSIC_SexyContent\SexyContent\Installer\InstallationController.cs:line 230
   at ToSic.SexyContent.WebApi.View.ModuleController.FinishInstallation() in C:\Projects\2SexyContent\Web\DesktopModules\ToSIC_SexyContent\Sxc WebApi\View\ModuleController.cs:line 260
   at lambda_method(Closure , Object , Object[] )
   at System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass10.b__9(Object instance, Object[] methodParameters)
   at System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Tracing.ITraceWriterExtensions.d__18`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Controllers.ApiControllerActionInvoker.d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Tracing.ITraceWriterExtensions.d__18`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Controllers.ActionFilterResult.d__2.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Filters.AuthorizationFilterAttribute.d__2.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Filters.AuthorizationFilterAttribute.d__2.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Controllers.ExceptionFilterResult.d__0.MoveNext()
```

**Comment by iJungleboy** [comment]
@Tychodewaard is this a confidential system, or could we have a copy of web + db to run it ourselves?

**Comment by iJungleboy** [comment]
...the copy would be the web before the upgrade...

**Comment by Tychodewaard** [comment]
You can have a copy as it is not in production, yet.
To what mail address can I wetransfer?

**Comment by iJungleboy** [comment]
info 2sxc org

**Comment by Tychodewaard** [comment]
You got mail 😄

**Comment by iJungleboy** [comment]
ok, we'll look into it, thx.

**Comment by raphael-m** [comment]
Ok, I was able to setup the copy and reproduce the issue. 2sxc logs installation / upgrade errors in the folder `\DesktopModules\ToSIC_SexyContent\Upgrade\Log`. In your case 2sxc logged that it was not installed successfully; the following error message is logged:

    The type initializer for 'ExtentPlaceholderCreator' threw an exception

Now this led me to an issue we stumbled over a few months ago, which is caused by having Evotiva-DNNBackup and 2sxc installed side-by-side: https://github.com/2sic/2sxc/issues/1136#issuecomment-289783130.

Since you have no running instances of 2sxc on your site, the easiest solution is to uninstall 2sxc 8, and then install the latest version 9.2 which runs on Entity Framework Core and should not conflict with Evotiva-DNNBackup anymore. I have tested this with your site and it worked flawlessly.
